src/util/isValueEvent.js

```diff
@@ -0,0 +1,17 @@
+export default (value, events) => {
```

Comment:
No need to duplicate functionality of `isValueAllowed`, just use the existing function (probably method name could be changed to more generic, but that can be done later unless you can suggest something now)

Reply:
completed in e967154

Reply:
Hrm, what I meant is that you can just use `isAllowedDate` in the component: https://github.com/vuetifyjs/vuetify/pull/2676/files#diff-5f403996a38f48f5de2879d0f0471a28R72

`const isEvent = isValueAllowed(date)` instead of `const event = isValueEvent(date)`

There's no need to create a separate util that doubles the functionality of `isValueAllowed`, test is also not necessary as there is a test for `isValueAllowed`, sorry if it wasn't clear before